antora-ui-camel/src/css/docs.css

```diff
@@ -1,54 +1,70 @@
-.projects h2 {
+.docs h2 {
   text-align: center;
 }
 
-.projects .project {
+.camel-project {
+  width: calc(100% - 2.5rem);
   display: inline-flex;
   border: 1px solid var(--color-smoke-90);
-  width: 46%;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  padding: 1rem 1rem 2rem 1rem;
+  margin: 1rem;
+}
+
+.camel-project .project {
+  border: none;
+}
+
+.project {
+  display: inline-flex;
+  border: 1px solid var(--color-smoke-90);
+  width: calc(50% - 2.5rem);
```

Comment:
The width isn't calculated to be in the center of the page horizontally. Best set the width to
`width: calc(50% - 2.2rem)`